neuralmonkey/vocabulary.py

```diff
@@ -64,6 +64,28 @@ def from_file(path: str) -> 'Vocabulary':
     return vocabulary
 
 
+def from_wordlist(path: str, encoding: str="utf-8") -> 'Vocabulary':
+    """Loads vocabulary from a wordlist.
+
+    Arguments:
+        path: The path to the wordlist file
+        encoding: The encoding of the merge file (defaults to UTF-8)
+
+    Returns:
+        The new Vocabulary instance.
+    """
+    vocabulary = Vocabulary()
+
+    with open(path, encoding=encoding) as wordlist:
+        for line in wordlist:
+            vocabulary.add_word(line.strip())
```

Comment:
ještě bych tu dal podmínku, aby to přeskakovalo prázdný řádky, aby se do slovníku nedostalo prázdný slovo